app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  # def index
+  #     @posts = Post.all
+  # end
```

Comment:
- **[OPTIONAL]** You need to have an active Post index. Kindly uncomment this code